From d9243c3885a9116eb2010c79ebb67c513c9bca53 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 19 Jun 2018 13:22:07 +0800
Subject: [PATCH 4026/5242] MLK-18632-1 ARM64: dts: freescale: imx8dx: move mu
 interrupt controller to gic

commit  2749f05a4f393cbc09207e82cec002e69cd4055b from
https://source.codeaurora.org/external/imx/linux-imx.git

To support lowest power mode for suspend, if no wakeup source
from non-secure partition is enabled, IRQSTEER can be powered
off when suspend, so non-secure linux OS needs to pass WU
irqchip wakeup source info to ATF, as MU is always enabled
as wakeup source, and it is a system level resource, so no
need to have it in WU domain.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Bai Ping <ping.bai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index bd43b4b..c5f5310 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -129,7 +129,7 @@
 		compatible = "fsl,imx8-mu";
 		reg = <0x0 0x5d1c0000 0x0 0x10000>;
 		interrupts = <GIC_SPI 177 IRQ_TYPE_LEVEL_HIGH>;
-		interrupt-parent = <&wu>;
+		interrupt-parent = <&gic>;
 		fsl,scu_ap_mu_id = <0>;
 		status = "okay";
 	};
-- 
1.7.9.5

